test/jdk/java/security/Policy/Root/Root.java

```diff
@@ -27,9 +27,13 @@
  * @summary User Policy Setting is not recognized on Netscape 6
  *          when invoked as root.
  * @library /test/lib
- * @run testng/othervm Root
+ * @run testng/othervm/manual Root
  */
 
+/*
+* Run test as root user.
```

Comment:
Can you print set of instructions to run test as root? Also it will help if test fails when not run as root to begin with.